File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-search-for-and-replace-text--in-documents_4.vb
    Private Sub SelectionFind()
        Dim findText As String = "find me"

        Application.Selection.Find.ClearFormatting()

        If Application.Selection.Find.Execute(findText) = True Then
            MessageBox.Show("Text found.")
        Else
            MessageBox.Show("The text could not be located.")
        End If
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-search-for-and-replace-text--in-documents_4.cs
        private void SelectionFind() 
        { 
            object findText = "find me";

            Application.Selection.Find.ClearFormatting();

            if (Application.Selection.Find.Execute(ref findText,
                ref missing, ref missing, ref missing, ref missing, ref missing, ref missing,
                ref missing, ref missing, ref missing, ref missing, ref missing, ref missing, 
                ref missing, ref missing)) 
            { 
                MessageBox.Show("Text found.");
            } 
            else
            { 
                MessageBox.Show("The text could not be located.");
            } 
        }